mediaplayerbackend.cpp

```diff
@@ -297,20 +306,22 @@ void MediaPlayerBackend::onStateReceived(mopidy::PlaybackState state)
 void MediaPlayerBackend::onPlaybackStateChanged(mopidy::PlaybackState oldState,
                                                 mopidy::PlaybackState newState)
 {
-    qDebug() << "onPlaybackStateChanged";
     switch (newState) {
     case mopidy::PlaybackState::Paused:
+        qDebug() << "onPlaybackStateChanged: Paused";
         m_playbackState = QIviMediaPlayer::Paused;
         m_playbackController.getCurrentTlTrack();
         break;
 
     case mopidy::PlaybackState::Playing:
+        qDebug() << "onPlaybackStateChanged: Playing";
         m_playbackState = QIviMediaPlayer::Playing;
         m_playbackController.getCurrentTlTrack();
         m_playbackController.getTimePosition();
         break;
 
     case mopidy::PlaybackState::Stopped:
+        qDebug() << "onPlaybackStateChanged: Stopped";
```

Comment:
pl add default value

Reply:
what would default case do?

Reply:
For example:
default:
  print qWarning() << "onPlaybackStateChanged: error, state not supported: " << state;
  break;